parkings/models/parking.py

```diff
@@ -203,6 +206,11 @@ def __str__(self):
     def archive(self):
         return self
 
+    def sanitize(self):
+        self.registration_number = sanitize_registration_number(self.registration_number)
+        self.sanitized_at = timezone.now()
+        self.save()
```

Comment:
Also sanitize the normalized registration number in `normalized_reg_num` field.

Add `update_fields=['registration_number', 'normalized_reg_num', 'sanitized_at']` argument to the save call.